distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -64,5 +64,14 @@ export EXTRA_JAVA_OPTS
 # export KARAF_DATA # Karaf data folder
 # export KARAF_BASE # Karaf base folder
 
-export KARAF_OPTS="-Dfile.encoding=UTF8 -Dddf.home=$DDF_HOME"
+#
+# Admins can uncomment the following line and comment out the other definition of
+# the KARAF_SYSTEM_OPTS variable in order to determine any missing security permissions
+# should they install third-party bundles requiring additional access.
+#
+# export KARAF_SYSTEM_OPTS="-Dprograde.generated.policy=$DDF_HOME/generated.policy -Dprograde.use.own.policy=true -Djava.security.manager=net.sourceforge.prograde.sm.PolicyFileGeneratorJSM -Djava.security.policy==$DDF_HOME/etc/default.policy -DproGrade.getPermissions.override=sun.rmi.server.LoaderHandler:loadClass,org.apache.jasper.compiler.JspRuntimeContext:initSecurity"
+
+export KARAF_SYSTEM_OPTS="-Djava.security.manager=net.sourceforge.prograde.sm.ProGradeJSM -Djava.security.policy==$DDF_HOME/etc/default.policy -DproGrade.getPermissions.override=sun.rmi.server.LoaderHandler:loadClass,org.apache.jasper.compiler.JspRuntimeContext:initSecurity"
```

Comment:
Using `KARAF_SYSTEM_OPTS` here and in the Windows script in order to provide differentiation between `client` and `ddf` scripts. There are other ways we could handle the different needs - the client script does _not_ need a security manager - but they would require managing more files separate from the Karaf distro.